examples/bar.jl

```diff
@@ -35,38 +34,32 @@ function bar4()
     data = bar(;x = ["Product A", "Product B", "Product C"],
                 y = [20, 14, 23],
                 text = ["27% market share","24% market share", "19% market share"],
-                marker_color = "rgb(158,202,225)",
-                marker_opacity = 0.6,
-                line_color = "rgb(8,48,107)",
-                line_width = 1.5)
+				marker = attr(color = "rgb(158,202,225)", opacity = 0.6),
```

Comment:
Good to see @caimichael has excellent taste with text editors.  I'm not sure why tabs are appearing in the file.  In my .vimrc I just have

```
set tabstop=4             " no. of spaces when press tab
set expandtab             " tabs expanded into spaces
```

and everything works fine.  It seems that you can convert tabs to spaces in one go using `:retab`.  See

http://vi.stackexchange.com/questions/495/how-to-replace-tabs-with-spaces